### PR TITLE
Repair the project structure.

### PR DESCRIPTION
**Pull request by asynts**
When I moved the code over from core I misunderstood the structure and made the application project depend on the infrastructure project, when it really should have been the other way around. 

@misha130 explained this to me in this thread: https://github.com/codidact/authentication/pull/9#discussion_r396579559 .

### CODE REVIEW

**Comment by asynts** [comment]
@Daksh14 found a small bug, I added a commit to fix it.

https://ptb.discordapp.com/channels/634104110131445811/635510829499940912/694959512536481882

**Comment by misha130** [comment]
Note to self in general: There are many modifiers that are public when they should just be internal and have no business being public.

File: src/Infrastructure/Infrastructure.csproj [hunk 1]
```diff
@@ -13,12 +13,12 @@
         <PackageReference Include="IdentityServer4.AspNetIdentity" Version="3.1.2" />
     </ItemGroup>
 
-    <ItemGroup>
-        <ProjectReference Include="../Domain/Domain.csproj" />
-        <ProjectReference Include="..\Application\Application.csproj" />
```
Comment (misha130) [inline src/Infrastructure/Infrastructure.csproj:18]: Should be / for consistency

Comment (asynts) [reply src/Infrastructure/Infrastructure.csproj:18]: Can you suggest? I do not have a development environment setup on Windows.

Comment (asynts) [reply src/Infrastructure/Infrastructure.csproj:18]: 6233f0f7372b2450a4e8913e2939f7ea8616ca24

**Pull request merged**